### ISSUE TITLE
"new GPUInternalError(...)" should return type GPUInternalError

### ISSUE THREAD

**Issue by mattsoulanille** [gpuweb/types#108]
[GPUInternalError's](https://github.com/gpuweb/types/blob/main/generated/index.d.ts#L2281) `new` function / constructor is defined without a return type, so it defaults to `any`. Should this return type be the `GPUInternalError` interface instead?
```typescript
interface GPUInternalError
  extends GPUError {
  /**
   * Nominal type branding.
   * https://github.com/microsoft/TypeScript/pull/33038
   * @internal
   */
  readonly __brand: "GPUInternalError";
}

declare var GPUInternalError: {
  prototype: GPUInternalError;
  new (
    message: string
  );
};
```

Proposed change:
```typescript
...
declare var GPUInternalError: {
  prototype: GPUInternalError;
  new (
    message: string
  ): GPUInternalError; // <--- This right here.
};
```

I'd send a PR, but if I understand correctly, this file is generated.

**Comment by kainino0x** [comment]
Thanks for the report!

There's an upstream bug here:
https://github.com/darionco/bikeshed-to-ts/issues/11

But we can also maintain this as a manual diff in the types. They're partially generated, but not fully. I think we already have this manual edit for several other constructors. I can go ahead and add it.

**Comment by kainino0x** [comment]
published 0.1.23
